challenges/headstorm-bec-jehillert/server/middleware.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable quotes */
+const express = require('express');
+
+// Parses json requests only; returns error if invalid json
+module.exports.validateJSON = function(req, res, next) {
+  // eslint-disable-next-line consistent-return
+  express.json('strict'/* consider only arrays/objects */)(req, res, err => {
+    if (err) {
+      console.error(err);
+      return res.sendStatus(400); // Bad request
+    }
+
+    console.log('...req.body valid JSON?  OK');
+    next();
+  });
+}
+
+// rejects if 'list' payload is not an array
+module.exports.validateAndParseList = function(req, res, next) {
```

Comment:
I like this error handler, is there a reason you left the console.log's in here?

Reply:
The logs were to help me debug (I get myself into trouble if I do back-end without baby-steps and lots of feedback), and to make the code more comprehensible on your end.  But everything seems to be working and the code is not too complicated, so I'm happy to take them out.